src/HS/ListingBundle/EventListener/ControllerEventSubscriber.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace HS\ListingBundle\EventListener;
+
+use Symfony\Component\HttpKernel\Event\FilterControllerEvent;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+* 
+*/
+class ControllerEventSubscriber implements EventSubscriberInterface
+{
+	
+	/**
+     * @param FilterControllerEvent $event
+     */
+    public function onKernelController(FilterControllerEvent $event)
+    {
+    	var_dump($event);
```

Comment:
Tip: You could use Symfony's dump() function instead of var_dump.

Reply:
Nice thanks!